dbld/bootstrap

```diff
@@ -6,5 +6,7 @@ cd /source
 ./autogen.sh
 
 cd /build
+docbook_path=$(find /usr/ | grep '/docbook\.xsl' | grep man)
+CONFIGURE_OPTS="${CONFIGURE_OPTS} --with-docbook=${docbook_path}"
```

Comment:
What about this instead:
```
find /usr/share/{xml,sgml} -path '*/manpages/docbook.xsl'
```
Also, if we are autodetecting this, we might as well add this to the configure script, so it would also work for those who don't use the docker images.